benchmarks/rigid_shear/time-dependent/run_all_models

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Run all models in this directory. Note that resolution 8 and ppd 96 requires 192-384 GB of RAM
+
+for stokes_degree in 2 3; do #2 3
+  for discontinuous_pressure in false; do # true
+    for refinement in 3 4 5 6; do
+      for particles_per_direction in 4 5 6 7 10 15 20 32 45 64 80; do # 4 5 6 7 10 15 20
+        for generator in "reference cell"; do
+        for interpolator in "cell average" "bilinear least squares"; do # cell average
+        for integrator in "rk2" "rk4"; do # "rk2" "rk4"
+
+        echo "subsection Discretization" > current.prm
+        echo "  set Stokes velocity polynomial degree = $stokes_degree" >> current.prm
+        echo "  set Use locally conservative discretization = $discontinuous_pressure" >> current.prm
+        echo "end" >> current.prm
+
+        number_of_particles=`echo "$particles_per_direction * $particles_per_direction * 2 ^ (2 * $refinement)" | bc -l`
+        #ppc=`expr $particles_per_direction \* $particles_per_direction`
+        #min_ppc=`expr $ppc / 2`
+        echo "subsection Postprocess" >> current.prm
+        echo "  subsection Particles" >> current.prm
+        echo "  set Number of particles = $number_of_particles" >> current.prm
+        echo "  set Particle generator name = $generator" >> current.prm
+        #echo "  set Minimum particles per cell = $min_ppc" >> current.prm
+        #echo "  set Minimum particles per cell = $min_ppc" >> current.prm
+        #echo "  set Load balancing strategy = add particles" >> current.prm
```

Comment:
Is this useful to have?

Reply:
I think @jdannberg is asking a valid question :-)